docs/technical-overview.md

```diff
@@ -0,0 +1,250 @@
+
+`/dashboard` — Dashboard page showing the current user and assigned roles.
+
+`/password` — Password Change page for logged-in users; calls `POST podiumuaa/api/account/change_password`.
+
+`/reset/request` — Password Reset page, calls `POST podiumuaa/api/account/reset_password/init` to send a password reset email.
+
+`/reset/finish` — New Password page that finalizes the password reset process with `POST podiumuaa/api/account/reset_password/finish` call.
+
+`/register` — Registration page, calls `POST podiumuaa/api/register` to create a new user.
+
+`/settings` — User profile settings page, allows a user to specify their phone number / work department / job function.
+
+`/verify` — Verification page, calls `GET podiumuaa/api/verify` and/or `GET podiumuaa/api/reverify` to confirm email after registration.
+
+
+#### Admin Pages
+
+`/admin/docs` — Swagger API docs.
+
+`/admin/audits` — Audits page that shows system event history; calls `GET podiumuaa/management/audits` to fetch and filter the even list;
+
+`/admin/pdm-configuration` — readonly backend Configuration page; calls `GET management/env` and `GET management/configprops` to get the current server environment variables and JVM/Spring options.
+
+`/admin/pdm-elasticsearch` — Elasticsearch Reindex page; designed to fix ES data/mapping issues with a single button click.
+
+`/admin/pdm-gateway` — active Gateway Routes page; calls `GET api/gateway/routes/` to fetch services registered in Eureka.
+
+`/admin/pdm-health` — Health Checks page, shows system info fetched via `GET podiumuaa/management/health`.
+
+`/admin/logs` — Logging Settings page fetches current server logging configuration via `GET management/logs` and allows an admin user to update these settings via `PUT management/logs`.
+
+`/admin/pdm-metrics` — readonly JVM Application Metrics page, uses `GET management/metrics` and `GET management/dump`.
+
+`/admin/user-management` — User Management page that shows a list of existing users and provides a way to edit/delete/create user entries.
+
+`/bbmri/organisation` — Organization Management page that provides a way to specify organization details, available request types and set permissions for organization members.
+
+
+#### Requests interface
+
+The requests page (`/requests`) is the primary functional screen of the Gateway Frontend.
+It is available to researchers, reviewers and organization coordinators.
+
+It consists of a number of sub-views:
+
+`/my-requests` / `my-organisations` / `my-reviews` — role-dependent Request Overview page; displays existing requests and their statuses.
+
+`/new` / `/edit` — New/Edit Request form; creates draft request items via `PUT api/requests` and submits them for review via `GET api/requests/{uuid}/submit`.
+
+`/detail/{request-guid}` — Request Detail view that displays the current review step and lists attached files.
+
+
+### Gateway Server
+
+Gateway server is a Java/Spring app that serves as the core component of the Podium system.
```

Comment:
It also serves as a gateway, based on Zuul and Ribbon, for accessing the different services (podiumuaa and gateway), which are discovered using Eureka. In principle rate limiting and load balancing are supported by these libraries, but they are not enabled at the moment.